### ISSUE TITLE
Goals and models

### ISSUE THREAD

**Issue by lukego** [pybee/paying-the-piper#18]
Can you take a poll on Github somehow? It occurs to me that you can come at funding open source from a bunch of different perspectives and I wonder which people here want to accomplish:
1. Create a project and get paid to work on it.
2. Create a project that other people will manage to be paid to work on.
3. Get paid to contribute to open source in my current job.
4. Get a new job working directly on open source.
5. Pay other people to work an a project I care about.
6. Create a project as the basis for a startup company.
7. Create a project as the basis for a non-profit organization.

If you know which category you are in then you can decide which role models to pay attention to e.g. Linux, MySQL, SQLite, Mozilla, PostgreSQL, nginx, Inktank, and so on.

**Comment by seiyria** [comment]
Perhaps, make a strawpoll and link it in the OP?

**Comment by nanuxbe** [comment]
What about neither of those.
What I do as open-source I do it on my own will. If some people want to _give_ me money for what I do, I'd like to be able to accept it but that's not the reason I'm taking part in this discussion.
I also understand that some projects (hence the people working on those projects) need funding to be sustainable and I'd like to take part in the discussion in order for those people to be able to find the money they need while keeping open-source free as in beer for everyone who _need_ open-source to learn/make a living/create other projects/find a passion/help them achieve something/... (<- fill in whatever reason other than making big bucks)

**Comment by xordoquy** [comment]
I do think that there are a lot of different goals here. It will be interesting though imply a lot of work to gather what has been said and summarize the all under different chapters for the various goals each may have.

**Comment by freakboy3742** [comment]
Agreed that there are many possible goals here, which is evident from some of the suggestions that have been made. 

My goal for this project is (1), plus the longer term maintenance associated with (1) - for example, Django is a well established project at this point, but funding maintenance is proving difficult. 

The type of product you're developing also matters - if it's web based, SaaS might be viable; but that wont work for a kernel module.
